scripts/zkai_d128_layerwise_comparator_target_gate.py

```
@@ -0,0 +1,763 @@
+
+import argparse
+import copy
+import csv
+import hashlib
+import importlib.util
+import io
+import json
+import pathlib
+import sys
+import tempfile
+from typing import Any, Callable
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)